src/editor/ImageViewer.js

```diff
@@ -121,7 +123,28 @@ define(function (require, exports, module) {
         // add dimensions and size
         this._naturalWidth = e.currentTarget.naturalWidth;
         this._naturalHeight = e.currentTarget.naturalHeight;
-        
+
+        var img = e.currentTarget;
+
+        var vibrant = new window.Vibrant(img);
```

Comment:
I really wish there was a way to get this to work using proper AMD module loading vs. having to use it on the global.  Did you see if you could fix their code so it would work with requirejs?